controllers/seedimage_controller.go

```diff
@@ -448,12 +448,6 @@ func (r *SeedImageReconciler) getRancherServerAddress(ctx context.Context) (stri
 
 func fillBuildImagePod(name, namespace, buildImg, baseImg, configMap string, pullPolicy corev1.PullPolicy, deadline int32) *corev1.Pod {
 	const volLim = 4 * 1024 * 1024 * 1024 // 4 GiB
```

Comment:
We could increase it to 6GiB (the bare minimum required with the previous setup) or 8GiB (to match the maximum size allowed with the previous setup)

Reply:
Yes that is, so something like twice the max base image size. The thing is agreeing on what we consider the max, I am fine with 4Gib, so in that case we should set the volume to 8GiB.